include/boost/detail/winapi/dll.hpp

```diff
@@ -18,83 +19,186 @@
 
 #if !defined( BOOST_USE_WINDOWS_H )
 extern "C" {
+
 namespace boost { namespace detail { namespace winapi {
+struct MEMORY_BASIC_INFORMATION64_ {
```

Comment:
Does Windows SDK define both these structures? Shouldn't there be just one structure defined differently for different bitness?